.github/workflows/op-succinct-docker-image-builder.yml

```
@@ -0,0 +1,61 @@
+---
+  # Workflow for automatically building and deploying docker images for kurtosis-cdk.
+  name: OP-Succinct Docker Image Builder
+  
+  on:
+    workflow_dispatch:
+      inputs:
+        agglayer-op-succinct-commit:
+          description: The agglayer/op-succinct branch or commit to build the image from
+          required: true
+  
+  env:
+    IMAGE_NAME: leovct/op-succinct-contract-deployer
+    RUST_VERSION: 1.84.1
+    FOUNDRY_VERSION: nightly
```

nightly might introduce breaking changes, I'd recommend to use stable instead

```suggestion
    FOUNDRY_VERSION: stable
```